frontend/model/contracts/manifests.json

```diff
@@ -1,8 +1,3 @@
 {
-  "manifests": {
-    "gi.contracts/chatroom": "21XWnNKjPmeTVSNfQZ2DWD7KY4Be55R737KXSu4DDZmdhTwUrZ",
-    "gi.contracts/group": "21XWnNXj2ryGCoQEWy2KrDXBkyThZdGSWtZLsETg78h6rrpmjt",
-    "gi.contracts/identity": "21XWnNXrKKtTmrMJZZbjUQUocAwtETsgE8Ri6kY3K9Thy5bvMx",
-    "gi.contracts/mailbox": "21XWnNKoSm7e2vnGJvoFSRB7heFTeg24FwDCkb6JAazkzhpNdY"
-  }
+  "manifests": {}
```

Comment:
This file shouldn't be deleted.

And this reminds me: since you are making modifications to the chatroom contract, and since we haven't yet released version `0.0.1` of the contracts, you can safely delete `contracts/0.0.1` and regenerate them using `grunt pin:0.0.1`.

Just make sure to run `grunt dev` at least once immediately before running `grunt pin`, see #1352 (just created) for details.

Reply:
I didn't touch or delete this file manually. Also not sure in which ways the chatroom contract is modified. I would have thought only the identity contract would be involved, or maybe also the group contracts because of some transitivity since groups are made of users.
Going to follow your instructions anyway, but I wish there would be some automated way to detect and perform the necessary steps.

Reply:
Got this error after deleting `contracts/0.0.1` and running `grunt dev`:
![image](https://user-images.githubusercontent.com/64228468/179349705-d8230146-fa1d-42a7-a36a-16c7ef43563a.png)

Reply:
Oops, I meant the identity contract, sorry. Let’s try to troubleshoot this over Slack?

Reply:
@snowteamer this was also marked resolved but doesn't seem to be?

Reply:
I think I did delete the `contracts/0.0.1` folder, then ran `grunt dev --no-chunks` followed by `grunt pin:0.0.1` before the latest commit

Reply:
@snowteamer this file isn't being generated, or it is but without the hashes. Could you please investigate?